operate/qa/integration-tests/src/test/java/io/camunda/operate/elasticsearch/FinishedImportingIT.java

```diff
@@ -304,6 +304,26 @@ final var record = generateRecord(ValueType.PROCESS_INSTANCE, "8.6.0", 1);
         .until(() -> isRecordReaderIsCompleted("1-process-instance"));
   }
 
+  @Test
+  public void shouldWriteDefaultEmptyDefaultImportPositionDocumentsOnRecordReaderStart() {
```

Comment:
:x: I would like to see also an test where it doesnt resets the positions to zero again.

E.g. you have importer running it has set value x. Stop. Start again the same position should be persisted. 

Additionally, we could verify for 8.6 not all position document were existing but with 8.7 they are. We can do it in a follow up update test i guess.